Formula/spdx-sbom-generator.rb

```diff
@@ -0,0 +1,35 @@
+class SpdxSbomGenerator < Formula
+  desc "Support CI generation of SBOMs via golang tooling"
+  homepage "https://github.com/spdx/spdx-sbom-generator"
+  url "https://github.com/spdx/spdx-sbom-generator/archive/refs/tags/v0.0.13.tar.gz"
+  sha256 "7d088f136a53d1f608b1941362c568d78cc6279df9c1bdb3516de075cb7f10c3"
+  license any_of: ["Apache-2.0", "CC-BY-4.0"]
+
+  depends_on "go" => :build
+  depends_on "python@3.8" => :test
```

Comment:
Can we use `python@3.9` here so that we can use the `pipenv` formula?

Reply:
Unfortunately no bc the project Pipfile requires python3.8 so it fails when you use 3.9...

Reply:
Can we modify the Pipfile to make it use 3.9 instead? Or, alternatively, use a different project? Or, possibly even simpler: let's try to use `spdx-sbom-generator` on a non-existent project and just check we get the expected error message and exit code.

Reply:
So we can just run it on the test directory and we can check the error. I thought that we wanted to verify functionality though?